### DISCUSSION TITLE
[Feature Request] Ability to configure pac profile to use based on current folder

### DISCUSSION THREAD

**Discussion by filcole** [microsoft/powerplatform-vscode#553] [Ideas]
When using `pac solution unpack` or `pac solution export` verbs one has to remember to switch pac profile before using `pac solution unpack` or `pac solution clone`.

It's often that we are working on multiple solutions each already checked out from source control into a local folder.  When working on different solutions across different environments (e.g. one unmanaged solution per environment) it would be nice if there was a way to configure pac on my machine to use a specific profile for a specific folder by default. 

`pac profile --folder <folderpath> --useProfile <profileName>`

**Comment by Laskewitz** [comment]
Would be great to have a parameter for this with the `pac solution clone` command as well, for instance `pac solution clone --save-auth-profile` to remember the auth profile for that folder.
